modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -130,17 +130,27 @@ Do not apply any expiration rules. Delete all versions after 30 days.
 'versions_retention_obligation' => '30, 30',
 ----
 
-== Show the Author of a Version
+== Restoring of a Version
 
-You can enable and disable to show the author of a version in the versions history via an entry in config.php. For details see: xref:configuration/server/config_sample_php_parameters.adoc#save-and-display-the-author-of-each-version-of-uploaded-and-edited-files[Save and display the author of each version of uploaded and edited files]. This is helpful to keep track of who changed a version. 
+In the WebUI in the details view, there is a restore button next to the versions. It can be used to restore file to its past version. Restore does not delete old version, instead newer version is created. 
 
-After adding the author, the version screen changes in the following way:
+WARNING: Note that retention policy can be applied after restore operation, which could result in the old version be removed via retention policy. This behaviour can be changed with persistent major version discussed in the section below.
 
-image::configuration/files/files_versions/concious-persistent-major-versions.png[Author of a changed saved file,width=250]
+== Extended Version Metadata
+
+You can enable the feature via an entry in config.php. For details see: xref:configuration/server/config_sample_php_parameters.adoc#save-additional-metadata-for-versions[Save additional metadata for versions]. 
+
+image::configuration/files/files_versions/version-metadata.png[Extended Version Metadata,width=300]
+
+=== Show the Author of a Version
+
+You can enable and disable to show the author of a version in the versions history. This is helpful to keep track of who changed a version. 
 
 === Conscious Persistent Major Versions
 
-ownCloud can automatically generate version numbers for file versions. To enable this feature, the config setting to show a version's author must be enabled as described above. Based on this feature, one can define the actual version to be a major version which gets a major version number and all new upcoming versions start counting from this major number. To do so, the user has to click on the check mark on the left of the actual version to publish it as a major version.
+ownCloud can automatically generate version numbers for file versions. Based on this feature, one can define the actual version to be a major version which gets a major version number and all new upcoming versions start counting from this major number. To do so, the user has to click on the check mark on the left of the actual version to publish it as a major version. Major versions are persistent versions that are not subject to any retention policies set by ownCloud. Publishing the version persists it until the original file is deleted. 
```

Comment:
Is the  check mark really on the left?

Reply:
I have created a build and pushed it to staging for "optical" review.

Reply:
`the user has to click on the check mark on the left of the actual version to publish it as a major version` -> `the user has to click on the check mark next to the version info to publish it as a major version`

the checkmark is on right side...

Reply:
fixed by using the suggested text